releases/v20.1.0.md

```diff
@@ -0,0 +1,130 @@
+**RBAC Changes** | All [role-based access control (RBAC) features](../v20.1/authorization.html#roles) (`CREATE ROLE`, `ALTER ROLE`, `DROP ROLE`, `GRANT ROLE`, `REVOKE ROLE`) are now covered by the [BSL license](../v20.1/licensing-faqs.html) and available to non-enterprise users.
+**Security Updates** | Several security features have been added to CockroachDB v20.1, including the ability to [customize the mapping between TLS certificates and principals](../v20.1/create-security-certificates-openssl.html#step-2-create-the-certificate-and-key-pairs-for-nodes), to [name principals (users/roles) with periods](../v20.1//create-user.html#considerations) so as to reflect the structure of internet domain names, to [allow or disallow users from authenticating](../v20.1/create-user.html#set-login-privileges-for-a-user), and to [allow or disallow users from creating, altering, and dropping other users](../v20.1/create-user.html#allow-the-user-to-create-other-users). Also, CockroachDB's support for the PostgreSQL Host-Based Authentication (HBA) configuration language, which enables sites to customize the principal/client address/authentication method matrix, has been extended and unified.
+**`SELECT FOR UPDATE`** | The new [`SELECT FOR UPDATE`](../v20.1/select-for-update.html) statement lets you order transactions by controlling concurrent access to one or more rows of a table. It works by locking the rows returned by a selection query, such that other transactions attempting to `SELECT` the same data and then `UPDATE` the results of that selection are forced to wait for the transaction that locked the rows to finish. This prevents [transaction retries](../v20.1/transactions.html#transaction-retries) that would otherwise occur and, thus, leads to increased throughput and decreased tail latency for contended operations.
+**Nested Transactions** | CockroachDB now supports the nesting of transactions using savepoints. These [nested transactions](../v20.1/transactions.html#nested-transactions), also known as sub-transactions, can be rolled back without discarding the state of the entire surrounding transaction. This can be useful in applications that abstract database access using an application development framework or ORM. Different components of the application can operate on different sub-transactions without having to know about each others' internal operations, while trusting that the database will maintain isolation between sub-transactions and preserve data integrity.
+**Savepoint Naming** |  Savepoints for [client-side transaction retries](../v20.1/advanced-client-side-transaction-retries.html) default to using the name `cockroach_restart`, but the [savepoint name can now be customized](../v20.1/savepoint.html#customizing-the-retry-savepoint-name) to support applications that require different naming. For example, this may be necessary when using an ORM.
```

Comment:
This section can be removed.  We added the ability to customize the savepoint name (for "retry savepoints" only) in 19.1:

https://www.cockroachlabs.com/docs/v19.1/advanced-client-side-transaction-retries.html#customizing-the-savepoint-name